net/ola/Makefile

```diff
@@ -1,6 +1,6 @@
 #
-# Copyright (C) 2006-2015 OpenWrt.org
-# Copyright (C) 2015 Christian Beier <dontmind@freeshell.org>
+# Copyright (C) 2006-2017 OpenWrt.org
```

Comment:
small detail, don't update OpenWrt.org copyright as you are not affiliated with it (or are you?)